applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.h

```diff
@@ -203,6 +203,9 @@ class IncompressiblePotentialFlowElement : public Element
     void PrintData(std::ostream& rOStream) const override;
 
     ///@}
+protected:
+
+    void GetPotentialOnNormalElement(array_1d<double, NumNodes>& phis) const;
```

Comment:
Just for the future, when https://github.com/KratosMultiphysics/Kratos/pull/4682 is merged into master you can call this function from the utilities and avoid defining it as protected.